### PR TITLE
Updated Score property to the search results

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs [hunk 1]
```diff
@@ -93,6 +93,7 @@ public void IndexTestAutomapperAndTotalMessage(string searchTerm, SearchWidgetPa
                         ResultItemOverview = dummyIndexItem.ResultItem.Overview,
                         ResultItemUrlName = $"{defaultJobProfilePage}{dummyIndexItem.ResultItem.UrlName}",
                         Rank = (int)dummyIndexItem.Rank,
+                        Score = dummySearchResult.Results.First().Score.ToString(),
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: why not 
```suggestion
                         Score = dummyIndexItem.Score,
```

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs [hunk 2]
```diff
@@ -219,6 +220,7 @@ public void SearchResultsPaginationViewModelTest(string searchTerm, SearchWidget
...
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]:
```suggestion
                         Score = dummyIndexItem.Score,
```

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Models/JobProfileSearchResultItemViewModel.cs [hunk 3]
```diff
@@ -17,5 +17,7 @@ public class JobProfileSearchResultItemViewModel
         public string ResultItemUrlName { get; set; }
 
         public IEnumerable<string> JobProfileCategoriesWithUrl { get; set; }
+
+        public string Score { get; set; }
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Models/JobProfileSearchResultItemViewModel.cs]: why string?
```suggestion
         public double Score { get; set; }
```

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs (hunk 1 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: done this change

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs (hunk 2 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: done this change

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Models/JobProfileSearchResultItemViewModel.cs (hunk 3 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Models/JobProfileSearchResultItemViewModel.cs]: done this change

File: DFC.Digital/DFC.Digital.Data/Model/JobProfileIndex.cs [hunk 4]
```diff
@@ -81,5 +81,7 @@ public class JobProfileIndex : IDigitalDataModel
 
         [IsSearchable]
         public string CareerPathAndProgression { get; set; }
+
+        public double Score { get; set; }
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Data/Model/JobProfileIndex.cs]: Why do we need the Score in the index?!!! 😕

**Pull request merged**